jenkins/templates/deploymentconfig.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.deployment }}
+---
+apiVersion: apps.openshift.io/v1 
+Kind: DeploymentConfig
+metadata:
+  annotations:
+  template.alpha.openshift.io/wait-for-ready: "true"
+  name: {{ .Values.appName }}
+labels:
+  app: {{ .Values.appName }}
+spec:
+  replicas: 1
+  selector:
+  name: {{ .Values.appName }}
+  strategy: {{- if .Values.persistence.enabled }}
+    type: Recreate 
+    {{ else }}
+    type: Rolling 
+    {{ end }}
+  template:
+    metadata:
+      name: {{ .Values.appName }}
+    spec:
+      containers:
+      - name: {{ .Values.appName }}
+        env: {{ if .Values.deployment.openshiftauth }}
+        - name: OPENSHIFT_ENABLE_OAUTH
+          value: ${ENABLE_OAUTH}
+        - name: OPENSHIFT_ENABLE_REDIRECT_PROMPT
+          value: "true" {{ end }}
+        - name: KUBERNETES_MASTER
+          value: https://kubernetes.default:443
+        - name: KUBERNETES_TRUST_CERTIFICATES
+          value: "true"
+        - name: JENKINS_SERVICE_NAME
+          value: {{ .Values.appName }}
+        - name: JNLP_SERVICE_NAME
+          value: {{ .Values.appName }}-jnlp
+        {{- range $key := .Values.deployment.env_vars }}
```

Comment:
looks like you might be missing a `{{- end }}` two lines below to match with this

Reply:
added :)